### PR TITLE
fix e2e

### PR DESCRIPTION
**Pull request by yann300**
re add e2e tests `url`

### CODE REVIEW

File: apps/remix-ide-e2e/src/commands/openFile.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ class OpenFile extends EventEmitter {
 
 // click on fileExplorer can toggle it. We go through settings to be sure FE is open
 function openFile (browser: NightwatchBrowser, name: string, done: VoidFunction) {
-  browser.clickLaunchIcon('settings').clickLaunchIcon('filePanel')
+  browser.clickLaunchIcon('settings').pause(500).clickLaunchIcon('filePanel')
```
Comment (bunsenstraat) [inline apps/remix-ide-e2e/src/commands/openFile.ts:18]: why does openfile open settings first? do you know?

File: apps/remix-ide/ci/browser_tests_plugin_api.sh [hunk 2]
```diff
@@ -14,7 +14,7 @@ sleep 5
 
 npm run build:e2e
 
-TESTFILES=$(grep -IRiL "disabled" "dist/apps/remix-ide-e2e/src/tests" | grep "plugin_api" | sort | circleci tests split )
+TESTFILES=$(grep -IRiL "@disabled: true" "dist/apps/remix-ide-e2e/src/tests" | grep "plugin_api" | sort | circleci tests split )
```
Comment (bunsenstraat) [inline apps/remix-ide/ci/browser_tests_plugin_api.sh:17]: @disabled:( ?)true

maybe?

Comment (bunsenstraat) [inline apps/remix-ide/ci/browser_tests_plugin_api.sh:17]: "\'@disabled\': \?true"

Comment (bunsenstraat) [inline apps/remix-ide/ci/browser_tests_plugin_api.sh:17]:
```
"\'@disabled\': \?true"
```

File: apps/remix-ide-e2e/src/tests/stressEditor.test.ts [hunk 3]
```diff
@@ -0,0 +1,343 @@
...
+    init(browser, done)
+  },
+
+  'Should create 10 files, reload, and check if the files are saved': function (browser: NightwatchBrowser) {
+    const contents = {}
+    const checkContent = (i, done) => {
+      const name = 'test_' + i + '.sol'
+      browser
+        .openFile(name)
+        .pause(500)
```
Comment (bunsenstraat) [inline apps/remix-ide-e2e/src/tests/stressEditor.test.ts:18]: so openFile gets 500 and here again another 500

File: apps/remix-ide-e2e/src/commands/openFile.ts [hunk 4]
```diff
@@ -15,13 +15,26 @@ class OpenFile extends EventEmitter {
...
-  browser.clickLaunchIcon('settings').clickLaunchIcon('filePanel')
-    .waitForElementVisible('li[data-id="treeViewLitreeViewItem' + name + '"', 60000)
-    .click('li[data-id="treeViewLitreeViewItem' + name + '"')
-    .pause(2000)
-    .perform(() => {
-      done()
-    })
+  browser.perform((done) => {
+    browser.isVisible('[data-id="remixIdeSidePanel"]', (result) => {
+      console.log(result)
```
Comment (bunsenstraat) [inline apps/remix-ide-e2e/src/commands/openFile.ts:20]: not needed right?

Comment (bunsenstraat) [reply apps/remix-ide-e2e/src/commands/openFile.ts:20]: I mean the console

Comment (yann300) [reply apps/remix-ide-e2e/src/commands/openFile.ts]: removed

**Review by bunsenstraat — APPROVED** [review]

**Pull request merged**